src/bundle/Resources/views/form_fields.html.twig

```diff
@@ -29,13 +29,38 @@
                 {{- block('content_type_choice_widget_options') -}}
             </div>
         {%- else -%}
-            <div class="ez-instant-filter__group-item"">
+            <div class="ez-instant-filter__group-item">
                 {{ form_widget(form[choice.value]) }}
             </div>
         {%- endif -%}
     {%- endfor -%}
 {%- endblock -%}
 
+{%- block _search_content_types_widget -%}
```

Comment:
I'd move those definitions to separate file, you can later `{% use ... %}` new file here.